solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/RestaurantCellView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  SampleComponentView.swift
+//  DeliveryAppChallenge
+//
+//  Created by Rodrigo Borges on 04/02/22.
+//
+
+import UIKit
+
+struct RestaurantCellViewConfiguration {
+    let restaurantName: String
+    let restaurantType: String
+    let restaurantImage: String
```

Comment:
Uma boa prática é que se a sua classe já remete a `Restaurant`, você pode apenas especificar nas propriedades a função delas sem precisar do prefixo `restaurant`.
```suggestion
    let name: String
    let detail: String
    let icon: String
```